CNAME

```diff
@@ -1 +0,0 @@
-omemo.top
```

Comment:
That's why I wrote the comment. The problem is that I can't show you the preview on my github with that CNAME in there, because it doesn't compile then anymore for me. And unfortunately githhub didn't give me the option to exclude that from the commit :/ But it said that the maintainers can change the commits -- so I wondered if you have the possibility to exclude that somehow? I'll try again, now it looks like I can change something here again, which didn't work yesterday.